app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -1771,8 +1770,17 @@ private void showPlaybackProgress(final long progress, final long duration) {
         final int progressSeconds = (int) TimeUnit.MILLISECONDS.toSeconds(progress);
         final int durationSeconds = (int) TimeUnit.MILLISECONDS.toSeconds(duration);
         positionView.setMax(durationSeconds);
-        positionView.setProgressAnimated(progressSeconds);
-        detailPositionView.setText(Localization.getDurationString(progressSeconds));
+        // If there is no player inside fragment use animation, otherwise don't because
+        // it affects CPU
```

Comment:
I prefer line breaks at a logical place, i.e. start the next line at "otherwise". That's not really an issue though (so don't waste time changing it unless you happen to need to change something else in this PR as well). I'll take an actual look at the code later today (in like 2 hours).